code/rdb/datastripe.q

```diff
@@ -28,5 +28,4 @@ initdatastripe:{
     .ds.checksegid[];    
     };
 
-if[.ds.datastripe;.proc.addinitlist[(`initdatastripe;`)]];
```

Comment:
I'm a little confused why this doesn't work but the other way does? my understanding of the initlist in TorQ is that the functions in it will be run after all the scripts are loaded etc., so it should be running after the subscription anyway?

Reply:
So .ds.datastripe used to be set as default for rdb/wdb as on so it would then add it to the list, now it is default to 0 and then is set to on inside the subscription function. This means that the initdatastripe function was not being added to the init list

Reply:
Ahhh, of course, that makes sense. Cool, thanks for explaining :+1: